Formula/samba.rb

```diff
@@ -66,13 +66,19 @@ def install
       # macOS has its own SMB daemon as /usr/sbin/smbd, so rename our smbd to samba-dot-org-smbd to avoid conflict.
       # samba-dot-org-smbd is used by qemu.rb .
       mv "#{sbin}/smbd", "#{sbin}/samba-dot-org-smbd"
+      # Rename mdfind and profiles as well to avoid conflicting with /usr/bin/{mdfind,profiles}
+      mv "#{bin}/mdfind", "#{bin}/samba-dot-org-mdfind"
+      mv "#{bin}/profiles", "#{bin}/samba-dot-org-profiles"
```

Comment:
```suggestion
      %w[smbd mdfind profiles].each do |cmd|
        mv "#{sbin}/#{cmd}", "#{sbin}/samba-dot-org-#{cmd}"
      end
```

Reply:
smbd is in sbin, but other binaries are not.

Reply:
```suggestion
      # Rename mdfind and profiles as well to avoid conflicting with /usr/bin/{mdfind,profiles}
      { sbin => "smbd", bin => "mdfind", bin => "profiles" }.each do |dir, cmd|
        mv dir/cmd, dir/"sambda-dot-org-#{cmd}"
      end
```

Reply:
thanks, updated